.github/workflows/build_pico_sdk.yaml

```
@@ -27,54 +27,66 @@ on:
         default: examples_pico
         description: The path to the Pico SDK examples. Defaults to `examples_pico`.
 
+permissions: {}
 
 jobs:
   build:
     runs-on: ubuntu-latest
     env:
       # Customize the CMake build type here (Release, Debug, RelWithDebInfo, etc.)
-      BUILD_TYPE: Release
+      # this env var is supported by CMake as of v3.22+
+      CMAKE_BUILD_TYPE: Release
 
     steps:
-      - name: get repo info
+      - name: Get repo info
         id: lib-info
+        env:
+          REPO_NAME: ${{ github.event.repository.name }}
         run: |
-          lib_name=$(echo ${{ github.repository }} | sed 's;.\+/;;')
-          echo "name=$lib_name" >> $GITHUB_OUTPUT
+          echo "name=${REPO_NAME}" >> "${GITHUB_OUTPUT}"
           lib_deps=''
-          if [[ $lib_name == 'RF24Mesh' ]]; then
+          if [[ "${REPO_NAME}" == 'RF24Mesh' ]]; then
               lib_deps='RF24Network'
           fi
-          echo "deps=$lib_deps" >> $GITHUB_OUTPUT
+          echo "deps=${REPO_NAME}" >> "${GITHUB_OUTPUT}"
```

restore correct variable name for exporting to other steps
```suggestion
          echo "deps=${lib_deps}" >> "${GITHUB_OUTPUT}"
```